src/index.css

```diff
@@ -11,6 +11,10 @@ body {
   background-color: #ec448a;
```

Comment:
Please try to make almost an exact copy of the given design. Its a key part of this project and a major requirement to pass the task. 

Your design 
![image](https://user-images.githubusercontent.com/15807343/189962338-4957dae6-1290-4b8a-8fd4-2e66a1b3a66a.png)


Given design 
![image](https://user-images.githubusercontent.com/15807343/189962427-91e660f7-2f4f-41cd-aa2a-7ef4e9eb9301.png)

Have in mind that the task is to create a mobile web application so your styling should be created with those parameters 🤓